rllib/models/base_model.py

```diff
@@ -207,7 +207,6 @@ def _update_outputs_and_next_state(
         return outputs, next_state
 
 
-@Deprecated(error=False)
```

Comment:
Why are we removbing this deprecation warning? This is going to be deprecated soon according to our plans, no?

Reply:
Yes, absolutely. However, it is still active in many algos and thus users receive deprecation warnings for using non-deprecated algos. Also most algos (except PPO) still use the `_enable_learner_api=False` by default, so these warnings are too confusing.